openapi.yaml

```diff
@@ -16,175 +16,511 @@ host: api.oregonstate.edu
-        - in: query
-          name: species
-          description: Search by pet species
-          required: false
-          type: string
+        - name: gameID
+          in: path
+          description: unique integer
+          required: true
+          type: integer
       responses:
         '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/PetsResult'
+            $ref: '#/definitions/GameResult'
+        '404':
+          description: Game not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    put:
+      tags:
+        - games
+      summary: Update a game specified by its id
+      operationId: updateGameByID
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters: 
+        - name: gameID
+          in: path
+          description: ID of game to be updated
+          required: true
+          type: integer
+        - name: updateBody
+          in: body
+          description: Object that will overwrite the game specified by its id
+          required: true
+          schema:
+            $ref: '#/definitions/GameUpdateBody'
+      responses:
+        '200':
+          description: Successfully update a Game
+          schema:
+            $ref: '#/definitions/GameResult'
+          headers:
+            Location:
+              type: string
+              format: url
+              description: Location of the updated game
         '400':
-          description: Bad request
+          description: Invalid game object
           schema:
             $ref: '#/definitions/Errors'
         '500':
           description: Internal server error
           schema:
             $ref: '#/definitions/Errors'
-  /pets/{id}:
-    get:
-      summary: Info for a specific pet
+    delete:
       tags:
-        - pets
-      description: Get pet by unique ID
-      operationId: getPetById
+      - games
+      summary: "Remove a game specified by its id"
+      operationId: removeGameByID
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
       parameters:
+        - name: gameID
+          in: "path"
+          description: "Id of game to be removed"
+          required: true
+          type: "integer"
+      responses:
+        204:
+          description: "game successfully removed"
+        404:
+          description: "No game with this id"
+          schema:
+            $ref: "#/definitions/Error"
+        500:
+          description: "Internal server error"
+          schema:
+            $ref: "#/definitions/Error"
+  /games/{gameID}/players:
+    get:
+      summary: Get a list of players in the game
+      tags:
+        - players
+      description: get players by game id
+      operationId: getPlayersByGameID
```

Comment:
Nice, it's clear. Since you will two different kinds of IDs in your API (`GameId` and `PlayerId`), I'd suggest renaming the rest of operationId to specify which ID is using for the current method. For example:

1. `getPlayerByID` -> `getPlayerByPlayerId`
2. `removeGameByID` -> `removeGameByGameId`

and so on.